content/en/account_management/billing/google_cloud.md

```diff
@@ -5,15 +5,15 @@ kind: documentation
 
 ## Overview
 
-Datadog bills for hosts running the Agent and all GCE instances picked up by the Google Cloud integration. You are not billed twice if you are running the Agent on a GCE instance picked up by the Google Cloud integration.
+Datadog bills for hosts running the Agent and all GCE instances picked up by the Google Cloud integration. Services outside of GCE, such as [Dataflow][6], may create billable GCE hosts in Datadog. To see which hosts are billed, navigate to the GCE page in the Google Cloud console. You are not billed twice if you are running the Agent on a GCE instance picked up by the Google Cloud integration.
```

Comment:
Recommend being more specific here. "navigate to the GCE page" and look for what?

Reply:
Thanks for that feedback, I made some changes to clarify this.